systemd/ua-premium.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Ubuntu Advantage premium attach
+After=cloud-final.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ua attach-premium
+TimeoutSec=0
```

Comment:
What does this configuration do?

Reply:
Do you want to have this command retry for any reason? or are you fine with a single attempt?
For non-daemon services, the TimeoutSec controls how long systemd waits before marking success/failure for a service; in this case it's appropriate since once ua exits, we either are attached (success! return code of 0) or not attached (and return code non-zero).

This looks fine to me.  Let me know if there are other design goals here.